README.md

```diff
@@ -1,3 +1,55 @@
+
```

Comment:
I have started a directory, README/ which contains multiple README files about different topics. I then use the main README.md to link off to those topical readme files.

https://github.com/openpredictionmarkets/socialpredict/tree/main/README

Better formatting would be to start off with a new file called, "LOCAL_SETUP.md" and then put everything written in this merge request into that file.

From there, one could create a link to that LOCAL_SETUP.md with a description at this part of the bottom of the readme, so it acts as a sort of table of contents for sub-topics.

https://github.com/openpredictionmarkets/socialpredict/tree/main/README

Reply:
@pwdel great catch i will update the file with respect to your suggestion